integration-tests/pipelines/e2e-tests-staging-pipeline.yaml

```diff
@@ -228,7 +228,17 @@ spec:
                 [[ "${AFFECTED_PIPELINES}" == "release-pipelines" ]]; then
                 echo "This Test Suite is affected by changes in PR ${PR_NUMBER}"
   
-                "/home/e2e/tests/run-test.sh" "$(params.PIPELINE_TEST_SUITE)"
+                # Check if suite has a custom runner script
+                SUITE_DIR="/home/e2e/tests/$(params.PIPELINE_TEST_SUITE)"
+                CUSTOM_RUNNER="${SUITE_DIR}/run-idempotent-test.sh"
+                
+                if [ -f "${CUSTOM_RUNNER}" ] && [ -x "${CUSTOM_RUNNER}" ]; then
```

Comment:
But we want to run both I thought

Reply:
If so, I need to do another fix - for the UUID (it is the same for the second run).
Let me fix it and rerun